### PR TITLE
New package: Samsung.NetworkManager version 2.0.5.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

File: manifests/s/Samsung/NetworkManager/2.0.5.0/Samsung.NetworkManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.NetworkManager
+PackageVersion: 2.0.5.0
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=1830319&CDCttType=SW&ModelType=C&ModelName=NP-Q1U/P01/SEA&VPath=SW/200801/20080122135939796_Samsung_Network_Manager_2.0.5.0.zip
```
Comment (Trenly) [inline manifests/s/Samsung/NetworkManager/2.0.5.0/Samsung.NetworkManager.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/200801/20080122135939796_Samsung_Network_Manager_2.0.5.0.zip
```

File: manifests/s/Samsung/NetworkManager/2.0.5.0/Samsung.NetworkManager.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.NetworkManager
+PackageVersion: 2.0.5.0
```
Comment (Trenly) [inline manifests/s/Samsung/NetworkManager/2.0.5.0/Samsung.NetworkManager.installer.yaml:5]:
```suggestion
PackageVersion: 2.0
```

File: manifests/s/Samsung/NetworkManager/2.0.5.0/Samsung.NetworkManager.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.NetworkManager
+PackageVersion: 2.0.5.0
```
Comment (Trenly) [inline manifests/s/Samsung/NetworkManager/2.0.5.0/Samsung.NetworkManager.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0
```

File: manifests/s/Samsung/NetworkManager/2.0.5.0/Samsung.NetworkManager.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.NetworkManager
+PackageVersion: 2.0.5.0
```
Comment (Trenly) [inline manifests/s/Samsung/NetworkManager/2.0.5.0/Samsung.NetworkManager.yaml:5]:
```suggestion
PackageVersion: 2.0
```

**Review by Trenly — CHANGES_REQUESTED** [review]
> SI (s) (A4!18) [12:00:59:022]: Product: Samsung Network Manager 2.0 -- 1: The InstallScript engine is missing from this machine.  If available, please run ISScript.msi, or contact your support personnel for further assistance. 

> Property(S): STANDARD_USE_SETUPEXE = This installation cannot be run by directly launching the MSI package. You must run setup.exe.

**Comment by stephengillie** [comment]
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\Samsung\NetworkManager\2.0.5.0', expected 'manifests\s\Samsung\NetworkManager\2.0'

**Comment by upintheairsheep** [comment]
validation screenshot?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Standard error:  Installation failed with exit code -1978335226 
> 2023-07-25 16:27:52.125 [CLI ] ShellExecute installer failed: 1603 
> CustomAction ISMsiServerStartup returned actual error code 1603 but will be translated to success due to continue marking 1: ISStartUp Failure. OpenEvent, Error = 0x6  CustomAction ISStartup returned actual error code 1603 (note this may not be 100% accurate if translation happened inside sandbox) 1: RPC runtime reported exception 0x6ba  
> MSI (s) (F4:14) [16:27:51:793]: Product: Samsung Network Manager 2.0 -- Installation operation failed. 
> MSI (s) (F4:14) [16:27:51:793]: Windows Installer installed the product. Product Name: Samsung Network Manager 2.0. Product Version: 2.0.5.0. Product Language: 1033. Manufacturer: Samsung. Installation success or error status: 1603. 

image

**Comment by stephengillie** [comment]
The error `1603` sometimes means that a dependency is needed but not present. `VCRedist` 2015+ is by far our most common dependency, so it's probably that one.

**Comment by upintheairsheep** [comment]
> The error `1603` sometimes means that a dependency is needed but not present. `VCRedist` 2015+ is by far our most common dependency, so it's probably that one.

Add dependency tag

**Comment by Trenly** [comment]
Dependencies are now supported in the 1.6 client. If you could please submit a new manifest for the latest version of the package, it would be greatly appreciated!

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 12:52:39 PM: Product: Samsung Network Manager 2.0 -- 1: The InstallScript engine is missing from this machine.  If available, please run ISScript.msi, or contact your support personnel for further assistance.

(Automated response - build 513.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/s/Samsung/NetworkManager/2.0/Samsung.NetworkManager.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+PackageIdentifier: Samsung.NetworkManager
+PackageVersion: 2.0
+Installers:
+- InstallerUrl: https://downloadcenter.samsung.com/content/SW/200801/20080122135939796_Samsung_Network_Manager_2.0.5.0.zip
+  InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: msi
+  NestedInstallerFiles:
+  - RelativeFilePath: Samsung Network Manager 2.0.msi
```
Comment (stephengillie) [inline manifests/s/Samsung/NetworkManager/2.0/Samsung.NetworkManager.installer.yaml:13]:
```suggestion
  - RelativeFilePath: setup.exe
```

**Pull request closed without merging**